designsafe/static/scripts/projects/components/manage-experiments/manage-experiments.component.js

```diff
@@ -297,11 +297,11 @@ class ManageExperimentsCtrl {
             this.editExpForm = {};
             this.ui.showEditExperimentForm = false;
         }
```

Comment:
This might be outside the scope of this PR, but I'm noticing that when I add the first experiment or FR I need to exit the modal and re-enter it.  This doesn't happen when I'm adding my second experiment/FR.  Feel free to ignore.

Reply:
Interesting, thanks for testing that. I'm going to swing by later so you can show me what it's doing. Might need to make a Jira for that today.